pkg/ark-wallet/Makefile

```
@@ -0,0 +1,28 @@
+.PHONY: proto proto-lint vet run run-neutrino
```

```suggestion
.PHONY: vet run run-neutrino run-bitcoind run-neutrino-mutinynet
```

Let's also add a `make lint` command